tests/test_template_types.py

```diff
@@ -2411,6 +2411,14 @@ def test_letter_preview_template_lazy_loads_images():
     ]
 
 
+def test_broadcast_message_from_content():
```

Comment:
shouldn't the test be testing that we are getting a Template instance?

Reply:
That’s a good idea, makes the test a bit more robust, how about 26a58d8?

Reply:
nice one 🎉